applications/nrf_desktop/src/events/config_event.h

```diff
@@ -97,6 +97,7 @@ struct config_fetch_event {
 
 	u16_t recipient;
 	u8_t id;
+	bool usb;
```

Comment:
please use "void *channel_id" and pass address to config_channel instance - it's more transparent

Reply:
done